src/main/java/org/folio/rest/impl/PostOrdersHelper.java

```diff
@@ -49,13 +53,17 @@ public PostOrdersHelper(HttpClientInterface httpClient, Map<String, String> okap
         .thenApply(OrdersResourceImpl::verifyAndExtractBody)
         .thenAccept(poBody -> {
           PurchaseOrder po = poBody.mapTo(PurchaseOrder.class);
+          String poNumber = generatePoNumber();
```

Comment:
This works for the new PO and new PO lines case.  Do we need to take into account existing PO or PO lines, or a combination of both (exiting PO with new PO lines or additional (new) POL's)?